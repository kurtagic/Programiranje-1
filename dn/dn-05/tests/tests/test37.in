717
2 35
1 72
0 56
0 27
2 24
1 13
2 58
1 74
0 20
2 20
0 28
0 27
2 25
2 19
0 43
0 74
2 48
1 65
2 79
0 12
1 27
1 44
1 16
0 49
0 76
2 37
1 26
1 20
1 81
2 46
0 54
1 16
0 42
0 54
2 31
2 44
2 57
0 76
1 68
2 17
1 70
2 53
1 46
1 37
0 13
1 47
2 59
2 26
1 14
1 12
1 85
0 62
0 40
2 61
2 36
2 60
0 68
0 73
0 67
2 23
0 29
1 54
1 41
0 69
2 86
2 78
0 62
0 15
1 10
0 67
0 23
1 10
0 33
0 21
0 59
0 39
0 69
1 50
1 55
0 83
2 45
1 43
2 40
1 28
1 42
2 40
0 61
2 83
0 25
0 69
0 69
2 20
0 17
0 65
2 64
0 40
2 84
2 72
0 11
2 69
1 70
1 30
1 15
0 46
2 26
0 80
1 40
2 21
1 39
1 75
2 44
2 48
0 37
0 20
2 28
2 73
1 62
0 80
0 77
0 61
0 86
2 66
2 72
0 20
0 34
2 22
0 65
0 70
1 34
1 71
0 63
0 35
2 44
0 35
1 85
2 51
2 86
1 27
2 47
0 57
1 22
0 33
2 50
1 43
1 44
2 67
1 22
0 66
1 41
0 50
0 67
1 21
0 65
1 35
0 33
2 24
1 12
0 80
2 42
1 14
2 24
0 42
1 82
0 56
1 61
1 16
2 84
1 31
2 81
0 58
2 66
2 31
2 34
2 34
1 33
2 33
1 61
1 79
2 86
0 36
2 84
0 45
0 31
0 61
1 81
1 36
0 16
0 82
1 79
2 58
2 68
2 66
0 46
0 73
0 33
2 79
1 33
1 42
0 44
1 64
0 10
2 49
2 37
0 39
0 38
0 15
2 54
1 51
0 30
0 86
0 41
1 44
2 79
1 85
2 37
2 38
1 25
1 40
1 68
1 47
2 46
1 16
2 31
1 10
2 50
1 64
0 54
2 47
2 49
0 49
0 39
1 19
0 54
1 35
0 76
2 38
1 17
1 81
0 79
2 47
2 51
0 61
1 28
2 85
2 31
0 72
1 27
1 64
2 76
2 77
0 33
2 67
1 79
2 53
0 55
2 86
0 76
1 71
1 45
2 25
0 69
1 60
0 34
1 62
2 44
0 69
0 15
1 54
0 71
1 72
0 17
1 50
1 52
0 34
2 44
1 47
2 81
0 72
0 61
1 75
2 80
0 73
0 56
2 18
0 20
0 45
0 22
2 25
2 38
1 80
0 65
1 61
1 72
1 42
0 63
1 74
1 48
0 26
0 81
2 12
0 13
2 48
2 31
0 51
2 25
0 23
2 43
2 48
2 59
1 41
0 86
0 37
1 29
0 43
2 48
2 16
2 22
0 58
1 19
2 32
2 48
1 48
0 66
0 32
0 52
1 72
1 80
1 85
0 66
1 11
2 62
2 12
2 57
2 41
0 58
2 14
0 61
1 58
0 83
0 22
0 31
1 53
1 50
2 42
1 73
1 81
2 56
1 36
0 86
1 50
1 30
1 57
2 33
0 53
0 66
2 51
1 17
1 73
0 86
1 24
1 71
2 74
0 78
2 81
2 74
1 67
2 24
0 43
0 53
0 44
0 37
0 47
2 26
2 83
1 43
1 56
1 19
2 42
1 70
2 59
2 18
0 80
2 39
2 77
2 59
0 60
1 45
1 55
0 70
1 43
1 17
2 31
0 77
1 84
2 78
0 35
0 62
2 45
0 27
1 76
0 69
0 39
1 74
2 83
2 59
2 83
0 62
2 43
1 85
2 52
2 19
2 53
2 62
0 13
2 74
2 27
2 24
0 72
0 29
0 53
1 83
2 21
1 28
2 16
1 81
1 70
2 20
0 44
0 30
0 59
0 20
2 76
2 57
0 35
1 22
2 43
2 85
1 60
0 12
1 16
0 17
2 38
1 38
0 80
1 23
2 66
2 15
1 18
1 32
2 30
2 45
0 25
0 47
0 66
1 20
1 70
1 22
0 30
2 75
2 25
0 58
1 31
0 12
2 73
1 21
0 42
0 21
1 27
0 51
1 11
2 51
1 47
0 84
2 47
1 32
0 35
2 22
1 60
0 53
0 36
1 77
0 66
0 43
2 64
1 48
2 67
1 21
2 15
2 77
0 18
2 17
0 45
0 23
1 51
1 21
2 39
0 75
0 65
1 74
1 74
1 36
1 32
1 44
2 37
0 11
2 13
2 69
0 66
0 17
2 27
1 58
2 33
1 52
0 71
2 69
1 76
0 78
1 73
0 27
1 79
1 63
1 48
0 30
2 48
0 83
0 59
2 79
2 84
0 34
2 14
2 16
1 51
0 28
1 85
1 69
1 77
0 28
1 50
0 77
2 50
0 74
1 39
0 27
2 47
0 13
2 25
0 44
0 71
1 13
2 64
1 15
1 58
1 78
1 35
0 60
0 67
1 35
2 60
2 20
2 32
1 32
0 40
0 24
2 68
1 54
0 50
1 84
0 16
1 20
0 43
1 10
1 84
1 22
0 32
1 21
1 25
2 57
0 63
1 64
0 81
2 64
1 36
2 43
1 81
2 76
1 82
0 69
2 81
1 38
1 54
1 82
0 58
0 76
0 30
2 20
1 85
2 45
1 30
0 80
2 57
1 11
1 74
0 36
1 46
1 61
1 23
2 36
2 77
2 20
0 55
1 23
2 69
1 52
1 38
2 20
0 78
1 28
0 61
2 54
2 73
2 72
2 68
0 11
0 64
0 41
2 41
0 25
2 50
0 64
0 26
0 78
2 80
2 34
2 75
2 29
0 64
0 53
2 74
1 47
0 13
2 29
2 59
2 51
0 60
1 84
1 45
1 77
1 22
2 86
0 46
2 73
1 30
1 82
2 75
1 17
2 62
2 57
0 73
2 56
0 33
1 77
2 72
1 15
2 12
0 68
0 32
1 60
1 26
0 49
2 66
2 46
2 36
0 68
1 78
2 19
2 57
0 57
2 45
1 48
1 70
0 82
2 60
0 29
1 64
2 50
1 13
2 30
2 36
0 86
2 31
1 51
0 79
0 65
2 23
1 17
0 12
0 66
2 13
1 34
1 47
0 27
2 39
0 25
1 11
0 71
2 50
1 52
1 21
0 61
2 16
2 15
2 76
0 55
2 66
1 13
2 40
2 86
