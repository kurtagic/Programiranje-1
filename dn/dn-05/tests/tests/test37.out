10: 27
11: 73
12: 77
13: 83
14: 79
15: 79
16: 79
17: 75
18: 65
19: 76
20: 76
21: 78
22: 63
23: 80
24: 80
25: 99
26: 99
27: 99
28: 99
29: 98
30: 98
31: 99
32: 99
33: 85
34: 89
35: 85
36: 88
37: 88
38: 81
39: 79
40: 94
41: 94
42: 92
43: 90
44: 90
45: 98
46: 98
47: 100
48: 98
49: 95
50: 103
51: 103
52: 99
53: 80
54: 80
55: 86
56: 86
57: 86
58: 86
59: 85
60: 87
61: 88
62: 88
63: 88
64: 88
65: 85
66: 88
67: 88
68: 86
69: 86
70: 81
71: 84
72: 84
73: 84
74: 84
75: 81
76: 83
77: 83
78: 80
79: 80
80: 80
81: 80
82: 80
83: 79
84: 79
85: 79
86: 57
87: 57
88: 55
89: 55
